src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,352 @@
+            onScroll={onScroll}
+            sections={parseResultsIntoAll(dappList, selectedFilter, favoriteDappsById)}
+            renderItem={({ item: dapp }) => (
+              <DappCard
+                dapp={dapp}
+                section={DappSection.All}
+                onPressDapp={onSelectDapp}
+                onFavoriteDapp={onFavoriteDapp}
+              />
+            )}
+            keyExtractor={(dapp: DappV1 | DappV2) => dapp.id}
+            stickySectionHeadersEnabled={false}
+            testID="DAppsExplorerScreen/DappsList"
+            ListEmptyComponent={
+              <NoResults
+                filter={selectedFilter}
+                removeFilter={() => {
+                  setSelectedFilter({ id: 'all', name: t('dappsScreen.allDapps') })
+                  horizontalScrollView.current?.scrollTo({ x: 0, animated: true })
+                }}
+                testID="DAppsExplorerScreen"
+              />
+            }
+            ListFooterComponentStyle={{ flex: 1, justifyContent: 'flex-end' }}
+          />
+        )}
+      </>
+
+      {DappFavoritedToast}
+      {DappInfoBottomSheet}
+    </SafeAreaView>
+  )
+}
+
+function parseResultsIntoAll(
+  dappList: any,
+  filter: DappFilter,
+  favoriteDappsById: string[]
+): SectionData[] {
+  // Prevent favorite dapps from showing up in the all dapps section
+  const data =
+    filter.id === 'all'
+      ? dappList.filter((dapp: DappV2) => !favoriteDappsById.includes(dapp.id))
+      : dappList.filter(
+          (dapp: DappV2) =>
+            !favoriteDappsById.includes(dapp.id) &&
+            dapp.categories &&
+            dapp.categories.includes(filter.id)
+        )
+  // Return empty array if no results
+  if (data.length === 0) return []
+  // Else return dapps in all section
+  return [
+    {
+      data,
+      category: 'all' as unknown as CategoryWithDapps,
+    },
+  ]
+}
+
+function DescriptionView({ message, title }: { message: string; title: string }) {
+  return (
+    <View>
+      <Text style={styles.pageHeaderText}>{title}</Text>
+      <Text style={styles.pageHeaderSubText}>{message}</Text>
+    </View>
+  )
+}
+
+function DappFilterChip({ chipFilter, selectedFilter, setFilter, lastChip }: DappFilterChip) {
+  const filterPress = () => {
+    ValoraAnalytics.track(DappExplorerEvents.dapp_filter, { id: chipFilter.id })
+    setFilter(chipFilter)
+  }
+
+  return (
+    <View
+      style={[
+        styles.filterChipContainer,
+        // Filter chips color based on selected filter
+        chipFilter.id === selectedFilter.id
+          ? { backgroundColor: colors.onboardingBlue }
+          : { backgroundColor: colors.onboardingLightBlue },
+        // First Chip has slightly different margins
+        chipFilter.id === 'all'
+          ? { marginLeft: Spacing.Regular16 }
+          : { marginLeft: Spacing.Smallest8 },
+        // Last Chip has slightly different right margin
+        lastChip && { marginRight: Spacing.Regular16 },
```

Comment:
hmm i think we can simplify the styles a lot here, but it's hard to explain. i'll submit a PR and you can see if you prefer my suggestion

Reply:
what do you think about https://github.com/valora-inc/wallet/pull/3492?

Reply:
#3492 looks good to me! I left the key value, but otherwise made the changes in f17dfe22ae5c764d169170c818bfa012e6257f05.